api/v1beta1/conditions.go

```diff
@@ -60,15 +60,19 @@ const (
 )
 
 const (
-	// HeatRabbitMqTransportURLReadyCondition Status=True condition which indicates if the RabbitMQ TransportURLUrl is ready
+	// HeatRabbitMqTransportURLReadyCondition Status=True condition which indicates if the RabbitMQ TransportURL is ready
 	HeatRabbitMqTransportURLReadyCondition condition.Type = "HeatRabbitMqTransportURLReady"
+
+	// HeatRabbitMqNotificationURLReadyCondition Status=True condition which indicates if the RabbitMQ TransportURL is ready
+	HeatRabbitMqNotificationURLReadyCondition condition.Type = "HeatRabbitMqNotificationURLReady"
```

Comment:
`NotificationTransport` is more consistent with the actual parameter name, but that looks too long so I decided to use simple `Notification` here(and the other namings).